From 20ca15efb9e7547f9dd646a7dbbb90f1d1938cb3 Mon Sep 17 00:00:00 2001
From: Glenn Washburn <development@efficientek.com>
Date: Tue, 7 Dec 2021 23:36:31 -0600
Subject: [PATCH] configure: Fix misspelled variable BUILD_LDFAGS ->
 BUILD_LDFLAGS

Signed-off-by: Glenn Washburn <development@efficientek.com>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
(cherry picked from commit d17eddab81eab790689b00172a9ee8351548de82)
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 6ba21b180..43558b353 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1657,7 +1657,7 @@ CC="$BUILD_CC"
 CPP="$BUILD_CPP"
 CFLAGS="$BUILD_CFLAGS"
 CPPFLAGS="$BUILD_CPPFLAGS"
-LDFLAGS="$BUILD_LDFAGS"
+LDFLAGS="$BUILD_LDFLAGS"
 
 unset ac_cv_c_bigendian
 unset ac_cv_header_ft2build_h
-- 
2.25.1

